base/setup/lib/muifonts.h

```diff
@@ -304,6 +439,11 @@ MUI_SUBFONT HindiFonts[] =
 {
     /*Font                       Substitute  */
     { L"Arial",                  L"Liberation Sans" },
+    { L"Arial Baltic,186",       L"Arial,186" },
+    { L"Arial CE,238",           L"Arial,238" },
+    { L"Arial CYR,204",          L"Arial,204" },
+    { L"Arial Greek,161",        L"Arial,161" },
+    { L"Arial TUR,162",          L"Arial,162" },
     { L"Courier",                L"FreeMono" },
     { L"Courier New",            L"FreeMono" },
```

Comment:
Is Hindi missing the 'Courier New' block intentionally?

Reply:
Fixed.